tests/container_hardening/test_container_hardening.py

```diff
@@ -1,33 +1,45 @@
+import re
 import pytest
 import logging
-from tests.common.helpers.assertions import pytest_assert
+from tests.common.helpers.assertions import pytest_assert, pytest_require
+from tests.common.helpers.dut_utils import is_container_running, get_disabled_container_list
 
 pytestmark = [
     pytest.mark.topology('any'),
 ]
 
 logger = logging.getLogger(__name__)
 
-NO_PRIVILEGED_CONTAINERS = [
```

Comment:
>NO_PRIVILEGED_CONTAINERS [](http://example.com/codeflow?start=0&length=24)

Could you use a PRIVILEGED_CONTAINERS list? This list initial value may change in future.

Reply:
updated, not sure if it addresses what you're asking, please help re-review